src/app/components/mnoe-users-local-list/mnoe-users-local-list.coffee

```diff
@@ -78,6 +78,66 @@
           MnoErrorsHandler.processServerError(error)
       ).finally(-> user.isSendingInvite = false)
 
+    scope.availableRoles = () ->
+      [
+        "mnoe_admin_panel.constants.user_roles.member",
+        "mnoe_admin_panel.constants.user_roles.admin",
+        "mnoe_admin_panel.constants.user_roles.super_admin"
+      ]
+
+    scope.indexOfRoleInTable = (role) ->
```

Comment:
maybe something more like indexOfRoleInRoles or something like that? I find InTable confusing.